src/inference/sghmc.jl

```diff
@@ -58,6 +54,7 @@ end
 function step(model, spl::Sampler{<:SGHMC}, vi::VarInfo, is_first::Val{false})
     # Set parameters
     η, α = spl.alg.learning_rate, spl.alg.momentum_decay
+    spl.info[:eval_num] = 0
```

Comment:
Why are we resetting `eval_num` here?

Reply:
The original usage of `spl.info[:eval_num]` is to count how many times the model was executed for each `step()` call. But in fact these codes are removed now - I will add them back.